src/test/java/teammates/test/cases/ui/browsertests/BaseUiTestCase.java

```diff
@@ -20,6 +20,20 @@
 
 public class BaseUiTestCase extends BaseTestCase {
 
+    /** indicates if the test-run is to use GodMode */
+    protected static Boolean enableGodMode = false;
+
+    /**
+     * Checks if the current test-run should use godmode, 
+     * if yes, enables GodMode
+     */
+    @BeforeSuite
+    public static void checkAndEnableGodMode() {
+        if (enableGodMode) {
+            System.setProperty("godmode", "true");
+        }
+    }
+
```

Comment:
Yes it is the same. But this gives two benefits:
1. This allows us to use godmode when running a custom set of ui tests and not just one at a time or a predefined set.
2. This makes it easy for new developers to use godmode. They simply have to change the ```enableGodMode``` variable to true.